packages/mazeppa/mazeppa.0.3.3/opam

```diff
@@ -0,0 +1,57 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A modern supercompiler for call-by-value functional languages"
+maintainer: ["hirrolot <hirrolot@gmail.com>"]
+authors: ["hirrolot <hirrolot@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/mazeppa-dev/mazeppa"
+doc: "https://github.com/mazeppa-dev/mazeppa"
+bug-reports: "https://github.com/mazeppa-dev/mazeppa/issues"
+depends: [
+  "ocaml" {>= "5.1"}
+  "dune" {>= "3.15"}
+  "pprint"
+  "checked_oint" {>= "0.2.0"}
+  "ppx_deriving"
+  "ppx_string_interpolation"
+  "ppx_yojson_conv"
+  "ppx_blob" {>= "0.9.0"}
+  "bisect_ppx"
+  "menhir"
+  "spectrum" {>= "0.6.0"}
+  "clap"
+  "alcotest" {>= "1.7.0"}
+  "odoc" {with-doc}
+]
+available: arch != "arm32" & arch != "x86_32" & arch != "ppc64" & os != "win32" & switch != "5.3~alpha1" & os-family != "opensuse"
+x-ci-accept-failures: [
+  "fedora-38"
```

Comment:
Is it expected that this fails on all fedoras as well? Wouldn't it be better to check what the problem is?